Running cluster for deep fits
120110 120111 normal
Running object 120110
Not enough g observations!
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  44
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  62
Min gap is: 0.0078125
Number of observations now:  32
Counts: {'r': 41, 'i': 60, 'z': 24}
Mu: {'g': 0.0, 'r': 0.15114148, 'i': 0.23250769, 'z': 0.42171323}
No handles with labels found to put in legend.
Variable??
0
0
Non-Variable source in g
Variable??
0.02105357615215149
1.6632226003217592
Non-Variable source in r
Variable??
0.03462939139579652
0.8820310299071218
Non-Variable source in i
Variable??
0.08682151844515894
0.7538072056894635
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [1, 1, 0.6305427495420692, 1]
Slope Error: [0.0, 0, 0.14084691045969286, 0.5419838676809575]
intercepts:  [0, 0, 0.01157799294173937, -0.10621012709280095]
Int Error:   [0.0, 0, 0.033363424220641334, 0.07593207214386574]
StDev:       [0.0, 0.08560629, 0.049076173, 0.102082476]
Not variable, closing plots & going to next row
Successfully finished!
Rows 120110 to 120111 ran in 2.7532 seconds
